lib/exhal/document.ex

```diff
@@ -69,9 +69,14 @@ defmodule ExHal.Document do
   @doc """
     **Deprecated**
 
+    See to_json_map/1
+    """
+  def to_json_hash(doc), do: to_json_map(doc)
```

Comment:
But then you'll run out of major version integers at this rate! Not really worth bumping just to remove one delegated method.